public/file-repository.php

```diff
@@ -38,6 +38,10 @@
         $storagePath = $_ENV['FILE_STORAGE_PATH'];
     }
 
+    array_walk_recursive($_GET, function (&$input) {
```

Comment:
`array_walk_recursive` may be safely replaced with `array_walk`. We do not support multidimensional GET params

Reply:
Multi-dimensional GET parameters are not supported, but we cannot prevent users from transmitting. When multi-dimensional parameters appear, PHP Fatal error will appear when using array_walk

```
[error] 31#31: *206 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Uncaught TypeError: htmlspecialchars() expects parameter 1 to be string, array given in /webser/www/github-profile-views-counter/public/file-repository.php:42
Stack trace:
#0 /webser/www/github-profile-views-counter/public/file-repository.php(42): htmlspecialchars()
#1 [internal function]: {closure}()
#2 /webser/www/github-profile-views-counter/public/file-repository.php(43): array_walk()
#3 /webser/www/github-profile-views-counter/public/index.php(14): require('/webser/www/git...')
```